proxy/hdrs/MIME.h

```diff
@@ -283,13 +283,75 @@ struct MIMEHdrImpl : public HdrHeapObjImpl {
  *                                                                     *
  ***********************************************************************/
 
+/** A pre-parser used to extract MIME "lines" from raw input for further parsing.
+ *
+ * This maintains an internal line buffer which is used to keeping content between calls
+ * when the parse has not yet completed.
+ *
+ */
 struct MIMEScanner {
```

Comment:
Why don't you change it to class?

Reply:
Inertia. I'm changing so many things some of them slip by.